File: IOS_swif_study/TabBarController_No_storyboard_obj-c/TabBarController_No_Storyboard_objective-c/MyViewController.m
//
//  ViewController.m
//  TabBarController_No_Storyboard_objective-c
//
//  Created by Jeff Jeong on 2020/06/02.
//  Copyright © 2020 Tuentuenna. All rights reserved.
//

#import "MyViewController.h"

@interface MyViewController ()

@end

@implementation MyViewController


- (id) initWithParams:(NSString *)title bgColor:(UIColor *)bgColor {
    self.title = title;
    self.view.backgroundColor = bgColor;
    return self;
}

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}




@end

File: IOS_swif_study/14_TabBarController_No_storyboard_Swift/TabBarController_No_storyboard_Swift/MyViewController.swift
//
//  ViewController.swift
//  TabBarController_No_storyboard_Swift
//
//  Created by 성정현 on 2022/05/31.
//

import UIKit

class MyViewController: UITabBarController {
    
    convenience init(title: String, bgColor: UIColor){
        self.init()
        self.title = title
        self.view.backgroundColor = bgColor
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
    }

}
